plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/MultiParentCompileHook.java

```diff
@@ -28,6 +29,7 @@ public MultiParentCompileHook() {
 
 
     @Override
+    @SuppressFBWarnings(value = "REC_CATCH_EXCEPTION", justification = "silly rule")
```

Comment:
not informative. Is it for `} catch (Exception e) {` ?

Reply:
I was just following [existing](https://github.com/jenkinsci/workflow-api-plugin/blob/907cf64feb2f38e93aebbedf87946b0235c3dc93/src/main/java/org/jenkinsci/plugins/workflow/log/LogStorage.java#L125) [precedent](https://github.com/jenkinsci/workflow-durable-task-step-plugin/blob/00db760fa039f039cc63b5577a2e7b9f9d700981/src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java#L558) in the Jenkins codebase.